### DISCUSSION TITLE
The Shield Website

### DISCUSSION THREAD

**Discussion by kabugatti** [Th3-Shield/support#39] [General]
Sup folks 👋,
What's up with the website, I volunteered wanted to work on it but haven't gotten any heads up from anyone?

**Comment by brianfishers** [comment]
hello  @grand-rick001  we now need to work  on this shield project.   https://github.com/Th3-Shield/theshield-techsta   techsta.net is the live domain

**Reply by kabugatti** [reply]
Awesome. I noticed the `techsta.net` domain isn't working. Also, can I rewrite the website in Angular? It would be faster for me to contribute that way @toxicking-techguru

**Comment by brianfishers** [comment]
yes we are renewing within two days..... concerning angular the admins had agreed that we only use html,css,js(libraries not framework) ,css(framework), php  and  mysql  so that young developers can have easy time contributing. After one year we shall introduce other  JS framework.
